packages/wallet/config/paths.js

```diff
@@ -78,6 +79,7 @@ module.exports = {
   testsSetup: resolveModule(resolveApp, "src/setupTests"),
   proxySetup: resolveApp("src/setupProxy.js"),
   appNodeModules: resolveApp("node_modules"),
+  pureEVM: resolveApp("../../node_modules/pure-evm/pure-evm_bg.wasm"),
```

Comment:
Does this many any web project that includes `nitro-protocol` has to manually embed the `pure-evm` wasm? I'm wondering if we should remove `pure-evm` from `nitro-protocol` so it can be used with minimal configuration.

Reply:
I'm skeptical that `nitro-protocol` as a package should even be up for consideration to be a web-based dependency, let alone a publicly published one, but yeah I believe it does mean that. I agree removing it from `nitro-protocol` would be a good idea regardless though.